### ISSUE TITLE
Create frontend login page and redirect all wp-login.php to it

### ISSUE THREAD

**Issue by brashrebel** [realbig/RBM_TH_PDS#2]
Customers are getting confused and Wasentha isn't happy with the fact that the login page is our network wide default RBM branded wp-login.php page. We need to redirect all attempts to access that page to a new frontend login page instead. I think WooCommerce has login related shortcodes. Let's make an account page where users can login, register, edit their profile and see their purchase history.

**Comment by BigActual** [comment]
Did this occur?

**Comment by brashrebel** [comment]
No. I was discussing this as a small project with the client. The idea of looking very seriously at what the experience for customers should be and what information they should see. Wasentha needed to think about it and review my examples.

I think in the end we fixed a small bug and she just dropped it as a priority. I'd recommend looking into it again at some point.

**Comment by BigActual** [comment]
Yes, we're discussing it now. Thanks for clarifying.
